Faster Devcontainer

shadycuz opened the pull request:
This delays the install of additional pythons until after the container
starts.

shadycuz commented:
I didn't expect code coverage to drop just because we submitted coverage from multiple python versions. Kinda expected it to inscrease. 

@dhutchison Feel free to review and merge when you get a chance.

dhutchison commented:
I'll take a look at this evening (just heading out). 

That does sound weird, I'll take a bit of a dig in to it see what's going on.

shadycuz commented:
@dhutchison I had removed your "examples" folder. I did not know those were behaving as "tests" and were contributing so much to the code coverage. I have added them back =) 

Merge when you get a chance. I only changed a few things outside of dev container. 

I have tried to add support for 3.14 but its not working because of issue with pydantic because of taskcat.

dhutchison commented:
Yep that would do it - I think at the time I'd figured that if I needed to write examples for the documentation there wasn't much point duplicating that effort on other test cases. 

That's annoying about taskcat - I'll need to get around to a PR against it again for dependency versions (last time was a bit of a nightmare getting all their tests passing again, and I don't have any first hand experience using it at all)
